src/models/Employee.ts

```diff
@@ -3,7 +3,7 @@ import mongoose, { Document, Schema } from "mongoose";
 export interface IEmployee {
   name: string;
   email: string;
-  phone_number: string;
+  phoneNumber: string;
```

Comment:
for some reason Employee already inside master, so I need you to fix the interface like the comment on Department